app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx

```diff
@@ -101,7 +112,7 @@ const TabLabelCanSample = ({
         <div onClick={() => onExplore(entity)}>
           <Box className="wranglingHover">
             <WrangelIcon />
```

Comment:
Is this mispelled? should it be `WrangleIcon`?

Reply:
Yes, it was a typo error, corrected it